samples/snippets/src/main/java/com/google/cloud/examples/securitycenter/snippets/bigqueryexport/CreateBigQueryExport.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.examples.securitycenter.snippets.bigqueryexport;
+
+// [START securitycenter_create_bigquery_export]
+
+import com.google.cloud.securitycenter.v1.BigQueryExport;
+import com.google.cloud.securitycenter.v1.CreateBigQueryExportRequest;
+import com.google.cloud.securitycenter.v1.SecurityCenterClient;
+import java.io.IOException;
+import java.util.UUID;
+
+public class CreateBigQueryExport {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(Developer): Modify the following variable values.
+    // parent: Use any one of the following resource paths:
+    //              - organizations/{organization_id}
+    //              - folders/{folder_id}
+    //              - projects/{project_id}
+    // filter: Expression that defines the filter to apply across create/update events of findings.
+    // bigQueryDatasetId: The BigQuery dataset to write findings' updates to.
+    // bigQueryExportId: Unique identifier provided by the client.
+    // For more info, refer:
+    // https://cloud.google.com/security-command-center/docs/how-to-analyze-findings-in-big-query#export_findings_from_to
+    String parent = String.format("projects/%s", "your-google-cloud-project-id");
+    String filter =
+        "severity=\"LOW\" OR severity=\"MEDIUM\" AND "
+            + "category=\"Persistence: IAM Anomalous Grant\" AND "
+            + "-resource.type:\"compute\"";
+    String bigQueryDatasetId = "your-bigquery-dataset-id";
+    String bigQueryExportId = UUID.randomUUID().toString().toLowerCase();
+
+    createBigQueryExport(parent, filter, bigQueryDatasetId, bigQueryExportId);
+  }
+
+  // Create export configuration to export findings from a project to a BigQuery dataset.
```

Comment:
from a project --> from a project, folder, or organization

My understanding is that exports can be done at any one of these levels

https://cloud.google.com/security-command-center/docs/how-to-analyze-findings-in-big-query#set_up_a_new_export_to

Reply:
Thanks for the review, Catherine! 
The above main method contains a comment on the different types of parent resources that can be used.
Since I'm using only project in this method, the comment is specific. 

Do you propose changing it?